src/xenia/cpu/processor.h

```diff
@@ -46,10 +49,11 @@ enum class Irql : uint32_t {
 
 class Processor {
  public:
-  Processor(Memory* memory, ExportResolver* export_resolver,
+  Processor(Emulator* emulator, Memory* memory, ExportResolver* export_resolver,
```

Comment:
what's emulator used for here? It'd be nice not to have this dependency (as then we don't have to think about when emulator is/isn't present, etc)